docs/content/architecture/tablespaces.md

```diff
@@ -127,6 +127,58 @@ PVC to a Kubernetes Deployment causes the Pods in the deployment to restart.
 When the operation completes, the tablespace will be set up and accessible to
 use within the PostgreSQL cluster.
 
+## Removing Tablespaces
+
+Removing a tablespace is a nontrivial operation. PostgreSQL does not provide a
+`DROP TABLESPACE .. CASCADE` command that would drop any associated objects with
+a tablespace. Additionally, the PostgreSQL documentation covering the
+[`DROP TABLESPACE`](https://www.postgresql.org/docs/current/sql-droptablespace.html)
+command goes on to note:
+
+> A tablespace can only be dropped by its owner or a superuser. The tablespace
+> must be empty of all database objects before it can be dropped. It is possible
+> that objects in other databases might still reside in the tablespace even if
+> no objects in the current database are using the tablespace. Also, if the
+> tablespace is listed in the temp_tablespaces setting of any active session,
+> the DROP might fail due to temporary files residing in the tablespace.
+
+Because of this, and to avoid a situation where a PostgreSQL cluster is left in
+an inconsistent state due to trying to remove a tablespace, the PostgreSQL
+Operator does not provide any means to remove tablespaces automatically. If you
+do need to remove a tablespace from a PostgreSQL deployment, we recommend
+following this procedure:
+
+1. As a database administrator:
+  1. Log into the primary instance of your cluster.
```

Comment:
GitHub doesn't render this as a new list/ordering. Perhaps indent more?

Reply:
But hugo does.